src/main/java/com/hcl/appscan/cli/auth/LoginUtility.java

```diff
@@ -33,4 +40,31 @@ public static String getServer(String key) {
 		return SystemUtil.getServer((key == null ? "" : key));
 	}
 
+	public static String getClientType() {
+		String productName = CLIConstants.APPSCAN_CLOUD_CLI;
+		if (clientType == null) {
+			if (System.getenv().containsKey("CODEBUILD_CI") && System.getenv("CODEBUILD_CI").equals("true")) {
```

Comment:
Rather than specifically looking for an environment variable named "CODEBUILD_CI", we should probably make this a generic name that other tools that make use of the CLI could use. For example, any tool that uses the CLI could set the variable "APPSCAN_CLIENT". In the case of AWS CodeBuild, we would set:
APPSCAN_CLIENT=aws-codebuild
Similar for the version, we want to get the version of the client that's being used (in this case, the version of the AWS CodeBuild extension). We could use a generic variable name, such as "APPSCAN_CLIENT_VERSION" and each client can set that.
We should also sanitize the value to ensure that it only contains alphanumeric characters, dashes (-), dots (.), and/or underscores (_).

Reply:
Hi Matt , Got your point. For clarification do you mean that we need to explicitly set the env variables (APPSCAN_CLIENT and APPSCAN_CLIENT_VERSION ) from whichever client the CLI tool is being used from..
What if those env variables are not set by the user , do we need to set some default fallback values for those env variables ?

Reply:
Yes, it would be up to the individual clients to set those variables. If a client doesn't set the variables, we should fall back on a default to use the CLI tool and version information.

Reply:
Thanks Matt. I have made the changes as per these instructions. Please review the updated code. Thanks.